### ISSUE TITLE
Angular

### ISSUE THREAD

**Issue by AhmadNasary** [futurepress/epub.js#1307]
How to use the library in angular?
please help me!

**Comment by dzoibo** [comment]
what do you mean by use the library?

**Comment by AhmadNasary** [comment]
how to use ebub.js in nagular?

**Comment by CGarces** [comment]
Add `epub.min.js` to the scripts sections at angular.json

Declare and object and and use it.

```
declare let ePub: any;
...
const book = ePub(url || "https://s3.amazonaws.com/moby-dick/moby-dick.epub");
```

**Comment by AhmadNasary** [comment]
please send me all implementation code!

**Comment by AhmadNasary** [comment]
> Hey @AhmadNasary have you got the answer?

No! you can see any thing about epub.js implementation in angular

**Comment by CGarces** [comment]
> No! you can see any thing about epub.js implementation in angular

epub is a pure javascript library, use it in the same way than any other javascript library.
